src/main.c

```diff
@@ -2221,7 +2221,7 @@ int main(int argc, char **argv) {
         glfwSwapBuffers(window);
         glfwPollEvents();
         if (!typing && glfwGetKey(window, CRAFT_KEY_QUIT)) {
```

Comment:
I found it to be annoying as well. You could change the key binding to a key not close to the main ones, but this may be redundant with the exit button.